### DISCUSSION TITLE
2.0 alpha版本中如何使用任务结果传递功能

### DISCUSSION THREAD

**Discussion by fei126** [apache/dolphinscheduler#7034] [General]
2.0 alpha版本中如何使用任务结果传递功能，最新使用文档中未找到

**Comment by zhutinglei** [comment]
文档里有

https://dolphinscheduler.apache.org/en-us/docs/latest/user_doc/guide/parameter/context.html

下面这个例子是我的shell task，本质上是在日志中打印
```
${setValue(output="123")}
```
**注意**
- 这一行的开头必须是`${setValue(`，最后两个字符`)}`中间不能有空格
- 第一个节点定义OUT变量
- 下一个节点直接引用，不要定义同名局部变量

第一个节点
<img width="621" alt="Screen Shot 2021-12-02 at 11 21 18" src="https://user-images.githubusercontent.com/17734391/144351990-29128d0c-24ee-43d5-8759-5f2e9f4d8927.png">

第二个节点
<img width="615" alt="Screen Shot 2021-12-02 at 11 22 07" src="https://user-images.githubusercontent.com/17734391/144352060-92ffbc29-658b-4cb7-bb2e-97ef5f97fefd.png">

**Comment by zhongjiajie** [comment]
Hi @fei126 , to discuss or share your idea in community, please use English first for more general, thanks
